packages/fx-core/src/plugins/resource/frontend/dotnet/resources/errors.ts

```diff
@@ -65,6 +65,47 @@ export class ProjectPathError extends DotnetPluginError {
   }
 }
 
+export class UnknownScaffoldError extends FrontendPluginError {
```

Comment:
Is this possible to share the same error of frontend plugin instead of create the similar one?